client_test.go

```diff
@@ -78,6 +79,7 @@ func TestClientDoesntCachePartitionsForTopicsWithErrors(t *testing.T) {
 	seedBroker.Returns(metadataResponse)
 
 	config := NewConfig()
+	config.Net.KeepAlive = 12 * time.Millisecond
```

Comment:
Not sure whether this is a useful addition given that we don't actually test it.